src/main/java/uk/gov/companieshouse/reconciliation/config/ComparisonGroupConfig.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.companieshouse.reconciliation.config;
+
+public class ComparisonGroupConfig {
```

Comment:
Can we rename this class (e.g. ComparisonGroupModel), please? We already have a class called `ComparisonGroupConfiguration` used to configure `AggregationHandler` beans.